util/ssl/kafka-jaas.conf

```diff
@@ -0,0 +1,13 @@
+KafkaServer {
```

Comment:
Strikes me that a more natural location for this file is with the compose file that uses it?  See the overall review comment for something related.